packages/integration-tests/src/test-project-setup/conversation_handler_project.ts

```diff
@@ -51,6 +49,28 @@ type ConversationTurnAppSyncResponse = {
   content: string;
 };
 
+type ConversationMessage = {
```

Comment:
had to duplicate these for test purposes since we're hiding them in ai-constructs.